packages/suite/src/components/wallet/CoinjoinSummary/BalancePrivacyBreakdown.tsx

```diff
@@ -37,27 +35,39 @@ export const BalancePrivacyBreakdown = () => {
 
     const isSessionRunning = !!currentSession;
 
+    const getBalanceHeader = () => {
+        if (isSessionRunning) {
+            if (currentSession.paused) {
+                return 'Anonymisation paused';
```

Comment:
Has to be translatable

Reply:
77a4da176eb56e6e888f9ebe3d7a84d3f472a217